File: lib/data/app_contact.dart
class AppContact {
  final String name;
  final String phone;
  final String? uid;
  final String? photoUrl; // Base64 profile photo, same format as UserProfile.photoUrl
  final String source;

  const AppContact({
    required this.name,
    required this.phone,
    this.uid,
    this.photoUrl,
    this.source = 'UNKNOWN',
  });
}

File: android/app/src/main/kotlin/com/tbtrapp/data/AppContact.kt
package com.tbtrapp.data 

// A phone-book contact matched against the app's registered users
data class AppContact(
    val name: String,
    val phone: String,
    val uid: String? = null,
    val photoUrl: String? = null,   // Base64 profile photo, same format as UserProfile.photoUrl
    val source: String = "UNKNOWN"
)